gax-grpc/src/main/java/com/google/longrunning/OperationsClient.java

```diff
@@ -111,7 +109,6 @@
  * </code>
  * </pre>
  */
-@Generated("by GAPIC v0.0.5")
 @BetaApi
```

Comment:
Removed for this and OperationSettings. I left them in for the stub classes (since they're everywhere across the dependent APIs), but LMK if we should remove them in one fell swoop.